src/main/java/org/gooru/insights/constants/APIConstants.java

```diff
@@ -233,6 +235,8 @@
 	
 	public static final String DEFAULT_TO_MAIL = "default.to.email";
```

Comment:
@daniel1028 , DEFAULT_TO_MAIL field initialized by propery field. And as we are following the dot(".") as separator for all fields in property files, I have used that.